### PR TITLE
Add Cypress tests for the "complete security" phase

### PR DESCRIPTION
**Pull request by richvdh**
After you log in, there is a "complete security" phase which will get you to verify your device if possible.

There are no tests for this at all at the moment, so let's add some.

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: cypress/e2e/crypto/utils.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+/**
+ * wait for the given client to receive an incoming verification request
+ *
+ * @param cli - matrix client we expect to receive a request
+ */
+export function waitForVerificationRequest(cli: MatrixClient): Promise<VerificationRequest> {
+    return new Promise<VerificationRequest>((resolve) => {
+        const onVerificationRequestEvent = (request: VerificationRequest) => {
+            // @ts-ignore CryptoEvent is not exported to window.matrixcs; using the string value here
```
Comment (t3chguy) [inline cypress/e2e/crypto/utils.ts:31]: Looks exported to me

image

File: cypress/e2e/crypto/complete-security.spec.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        let botClient: CypressBot;
+        cy.getBot(homeserver, { displayName: "Jeff" })
+            .then(async (bot) => {
+                botClient = bot;
+                await bot.bootstrapCrossSigning({});
+            })
+            .then(() => {
+                // now log in, in Element. We go in through the login page because otherwise the device setup flow
+                // doesn't get triggered
+                console.log("%cAccount set up; logging in user", "font-weight: bold; font-size:x-large");
```
Comment (weeman1337) [inline cypress/e2e/crypto/complete-security.spec.ts:59]: I think we should use [Cypress' logging](https://docs.cypress.io/api/commands/log) here.

**Review by t3chguy — APPROVED** [review]
Looks sane otherwise

**Review by weeman1337 — APPROVED** [review]

File: cypress/e2e/crypto/complete-security.spec.ts (hunk 2 above)
Comment (richvdh) [reply cypress/e2e/crypto/complete-security.spec.ts:59]: No, I'm deliberately trying to put something in the javascript console here.

File: cypress/e2e/crypto/utils.ts (hunk 1 above)
Comment (richvdh) [reply cypress/e2e/crypto/utils.ts:31]: ugh but then I have to pass in `win` or `matrixcs` to `waitForVerificationRequest`.

This code isn't new, so I'm going to ignore this for now.

**Pull request merged**